tests/test_psbt_parser.py

```diff
@@ -1,8 +1,33 @@
 import pytest
 from mock import MagicMock
-from seedsigner.models import PSBTParser
+from seedsigner.models import PSBTParser, Seed
 from embit import psbt
+from embit import bip32
+from embit import bip39
+from binascii import a2b_base64, b2a_base64
+from seedsigner.models.settings_definition import SettingsConstants
 
-def test_sig_count():
-	print("test")
-	
\ No newline at end of file
+def test_p2tr_change_detection():
```

Comment:
We're only about 50% consistent at the moment, but ideally every test should have a docstring explaining what the test is confirming. I had previously tried to follow the "Should" convention (every test explanation starts with "Should...") but that ends up getting grammatically awkward.